src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,34 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.storage.StorageHandler;
+import duke.ui.UserInterface;
+
+import static duke.parser.Parser.parseCommand;
+
+public class Duke {
+    private StorageHandler storageHandler;
+    private UserInterface ui;
+
+    public Duke(String filePath){
+        storageHandler = new StorageHandler(filePath);
+        ui = new UserInterface();
+    }
+
+    public void run(){
+        String input = "empty";
+        Scanner in = new Scanner(System.in);
+        int isRunning = 1;
+
+
+        while(isRunning == 1){
+            input = in.nextLine();
+            isRunning = parseCommand(input);
```

Comment:
This doesn't read well. To the reader, it seems like you areo only parsing the command, and then deciding what to do based on the return value.
How to fix: 
- Need to find a more descriptive name for the method.
- Need to find a better place to execute the commands rather than in Parser. So code needs some reorganization.